corsikaio/subblocks/event_end.py

```diff
@@ -17,4 +19,44 @@
     Field(267, "n_em_particles_preshower"),
 ]
 
+event_end_fields_7xx = [
+    Field(1, "event_end", dtype="S4"),
+    Field(2, "event_number"),
+    Field(3, "n_photons_weighted"),
+    Field(4, "n_electrons_weighted"),
+    Field(5, "n_hadrons_weighted"),
+    Field(6, "n_muons_weighted"),
+    Field(7, "n_particles_written"),
+    Field(256, "longitudinal_fit_parameters", shape=6),
+    Field(262, "chi_square_longitudinal"),
+    Field(263, "n_photons_written"),
+    Field(264, "n_electrons_written"),
+    Field(265, "n_hadrons_written"),
+    Field(266, "n_muons_written"),
+    Field(267, "n_em_particles_preshower"),
+ ]
+
 event_end_dtype = build_dtype(event_end_fields)
+event_end_dtype_7xx = build_dtype(event_end_fields_7xx)
+
+def warn_dtype():
+    warnings.warn("Version unknown, using default event end definition dtype")
+    return event_end_dtype
+
+def warn_fields():
+    warnings.warn("Version unknown, using default event end fields definition")
+    return event_end_fields
+
+event_end_fields = defaultdict(warn_fields)
+event_end_fields[6.5] = event_end_fields
```

Comment:
And then you insert it into itself. This won't work.

A solution would be to call the variable above `event_end_fields_6xx`

Reply:
I understood what happened with tests, I was not using -e option in pip install, my bad